pager-indicators/src/main/java/com/google/accompanist/pager/PagerIndicator.kt

```diff
@@ -113,10 +113,14 @@ fun HorizontalPagerIndicator(
                     )
                 }
                 .size(width = indicatorWidth, height = indicatorHeight)
-                .background(
-                    color = activeColor,
-                    shape = indicatorShape,
-                )
+                .apply {
```

Comment:
Hey. It is wrong to use apply{} here unfortunately as Modifiers are build via the builder pattern, you always need to use the object returned by the next builder function and apply returns "this", not the new object. Instead you need something like
.then(if (pageCount > 0) Modifier.backround() else Modifier)